Frontend/CO.CDP.OrganisationApp.Tests/Registration/OrganisationIdentificationTests.cs

```diff
@@ -0,0 +1,105 @@
+﻿using CO.CDP.OrganisationApp.Pages.Registration;
+using CO.CDP.OrganisationApp.Tests.Common;
+using FluentAssertions;
+using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.AspNetCore.Mvc.RazorPages;
+using Microsoft.Extensions.Logging;
+using Moq;
+
+namespace CO.CDP.OrganisationApp.Tests.Registration
+{
+    public class OrganisationIdentificationModelTests
```

Comment:
Should add Test For Required parameter organisationIdentification, At the moment OnPost method is throwing exception.

Reply:
Updated and refactored the whole code.